apps/students/models.py

```diff
@@ -36,13 +39,16 @@ class Student(models.Model):
     remarks = models.TextField(max_length=5000, blank=True)
 
     def __str__(self):
-        return f'{self.get_full_name} ({self.school_class})'
+        return f'{self.get_full_name} ({self.school_class}) ({self.get_village}) ({self.get_profile_image_url})'
 
     @property
     def get_full_name(self):
         return f'{self.first_name} {self.last_name}'
 
     @property
+    def get_village(self):
+        return self.get_village_display()
+
```

Comment:
We don't really need this property. You can just use `student.get_village_display` in the template file, as you are now using `student.get_village`.

Reply:
done sir..